.github/workflows/ci.yml

```diff
@@ -1,7 +1,7 @@
 name: CI
 
 env:
-  SVDTOOLS_VERSION: 0.2.6
+  SVDTOOLS_VERSION: 0.2.7
```

Comment:
Note: this is not strictly needed to get the fix, but it does effectively bump the cache used by the CI (that I can't otherwise manually prune).

Reply:
fyi, you can clear caches now, see https://github.com/lpc-rs/lpc-pac/actions/caches